ImperatorToCK3Tests/ImperatorWorldTests/Countries/CountryTests.cpp

```diff
@@ -233,29 +234,29 @@ TEST(ImperatorWorld_CountryTests, color3DefaultsToNullopt)
 	input << "{\n";
 	input << "}";
 
-	const Imperator::Country theCountry(input, 42);
+	const auto theCountry = *Imperator::Country::Factory().getCountry(input, 42);
 
 	ASSERT_FALSE(theCountry.getColor3());
 }
 
 TEST(ImperatorWorld_CountryTests, correctCountryRankIsReturned)
 {
 	std::stringstream input;
-	const Imperator::Country theCountry1(input, 1);
-	
-	Imperator::Country theCountry2(input, 2);
+	auto theCountry1 = *Imperator::Country::Factory().getCountry(input, 1);
```

Comment:
If you're creating multiple countries in a test, it's worth considering storing the factory as a variable and using in each case.